exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -76,6 +76,9 @@ private ExecConstants() {
   public static final String SPILL_FILESYSTEM = "drill.exec.spill.fs";
   public static final String SPILL_DIRS = "drill.exec.spill.directories";
 
+  public static final String OUTPUT_BATCH_SIZE = "drill.exec.memory.operator.output_batch_sizeinMB";
```

Comment:
I wonder if MB is too coarse. Maybe just make this the batch size in bytes or K. If this was a config option, we could use HOCON syntax for MB and so on. Since it a session/system option, I suppose it must be a simple number. For example, below, in the external sort batch size, it is in bytes, isn't it?

Further, this should be restricted to be a system option. Check with Tim on how to do that; he and/or Jyothsna implemented a way to do that.